modules/claims_api/config/initializers/okcomputer.rb

```diff
@@ -75,6 +75,33 @@ def name
   end
 end
 
+class FaradayBGSCheck < BaseCheck
+  def initialize(service, arg)
+    @service = service
+    @arg = arg
+  end
```

Comment:
Can we set a default to `arg`, rather than declaring the default in every instance?
```class FaradayBGSCheck < BaseCheck
  def initialize(service, arg = '0')
    @service = service
    @arg = arg
  end```